content/fr/docs/reference/kubectl/quick-reference.md

```diff
@@ -0,0 +1,504 @@
+`-o=custom-columns-file=<filename>` | Affiche une table en utilisant le modèle de colonnes personnalisées dans le fichier  `<filename>`
+`-o=go-template=<template>`     | Affiche les champs définis dans un [golang template](https://pkg.go.dev/text/template)
+`-o=go-template-file=<filename>` | Affiche les champs définis par le [golang template](https://pkg.go.dev/text/template) dans le fichier `<filename>`
+`-o=json`     | Produit un objet API formaté en JSON
+`-o=jsonpath=<template>` | Affiche les champs définis dans une expression [jsonpath](/docs/reference/kubectl/jsonpath)
+`-o=jsonpath-file=<filename>` | Affiche les champs définis par l'expression [jsonpath](/docs/reference/kubectl/jsonpath)dans le fichier `<filename>` 
+`-o=name`     | Affiche uniquement le nom de la ressource et rien d'autre
+`-o=wide`     | Produit une sortie en format texte avec des informations supplémentaires, et pour les pods, le nom du noeud est inclus
+`-o=yaml`     | Produit un objet API formaté en YAML
+
+Exemples en utilisant `-o=custom-columns`:
+
+```bash
+# All images running in a cluster
+kubectl get pods -A -o=custom-columns='DATA:spec.containers[*].image'
+
+# All images running in namespace: default, grouped by Pod
+kubectl get pods --namespace default --output=custom-columns="NAME:.metadata.name,IMAGE:.spec.containers[*].image"
+
+ # All images excluding "registry.k8s.io/coredns:1.6.2"
+kubectl get pods -A -o=custom-columns='DATA:spec.containers[?(@.image!="registry.k8s.io/coredns:1.6.2")].image'
+
+# All fields under metadata regardless of name
+kubectl get pods -A -o=custom-columns='DATA:metadata.*'
+```
+
+Pour plus d'exemples, consultez la [documentation de référence](/docs/reference/kubectl/#custom-columns) de kubectl .
+
+### Verbositée et débogage des sorties de Kubectl
+
+La verbosité de Kubectl est contrôlée avec les options -v ou --v suivies d'un entier représentant le niveau de log. Les conventions générales de journalisation de Kubernetes et les niveaux de log associés sont décrits [ici](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md).
+
+Verbosity | Description
+--------------| -----------
+`--v=0` | Généralement utile pour que cela soit *toujours* visible pour un opérateur de cluster.
+`--v=1` | Un niveau de log par défaut raisonnable si vous ne souhaitez pas de verbosité.
+`--v=2` | Informations utiles sur l'état stable du service et messages de log importants pouvant correspondre à des changements significatifs dans le système. C'est le niveau de log par défaut recommandé pour la plupart des systèmes.
+`--v=3` | Informations supplémentaires sur les changements.
+`--v=4` | Verbosité de niveau débogage.
+`--v=5` | Verbosité de niveau trace.
+`--v=6` | Affiche les ressources demandées.
+`--v=7` | Affiche les en-têtes des requêtes HTTP.
+`--v=8` | Affiche le contenu des requêtes HTTP.
+`--v=9` | Affiche le contenu des requêtes HTTP sans troncature.
+
+## {{% heading "A suivre" %}}
```

Comment:
```suggestion
## {{% heading "whatsnext" %}}
```

Reply:
I've used the same pattern as the other French pages. It seems that almost all French pages use the same keyword, 'À suivre'.
what do you think ?

Reply:
To stay consistent, I think to continue to use 'À suivre'.